### DISCUSSION TITLE
shoka迁移到shokaX的方法，我是直接把ShokaX的git仓库客隆到本地还是按文档里面的办法迁移？

### DISCUSSION THREAD

**Discussion by Onsaki** [theme-shoka-x/hexo-theme-shokaX#352] [Q&A] [answered]
我迷茫在现在关于这个主题有两个文档，
一个是：https://github.com/theme-shoka-x/shokaX-docs
还有一个主题git仓库页面的：https://docs-hexo.shokax.top/
第一个文档写了怎么迁移用的好像是CLI，但是第二个链接的文档里面又说已经弃用CLI。
我不懂前后端，想知道我只是迁移的话用什么方法最简单快捷。
我的blog里面没多少数据，没有评论。

**Comment by D-Sketon** [comment]
推荐使用shokax-can

**Reply by Onsaki** [reply]
我看那个下载下来的文件好像是整个hexo加主题的，我是需要吧原来的站点文件直接覆盖进来嘛？

**Reply by D-Sketon** [reply, marked as answer]
参考文档迁移source吧，有些文件配置和文件名之类的和shoka不是完全相同

**Reply by Onsaki** [reply]
好的  谢谢🙏

**Reply by zkz098** [reply]
用SXC那个文档是旧文档，目前已经废弃了，如果后续有问题看shokax.top那个文档
